hbase-server/src/test/java/org/apache/hadoop/hbase/regionserver/TestEncryptionKeyRotation.java

```diff
@@ -91,6 +92,7 @@ public static void setUp() throws Exception {
     conf.setInt("hfile.format.version", 3);
     conf.set(HConstants.CRYPTO_KEYPROVIDER_CONF_KEY, KeyProviderForTesting.class.getName());
     conf.set(HConstants.CRYPTO_MASTERKEY_NAME_CONF_KEY, "hbase");
+    conf.setInt("hbase.hfile.compaction.discharger.interval", 10 * 60 * 1000);
```

Comment:
Why this config change?

Reply:
@infraio Thanks for reviewing.

> Why this config change?

I changed this to let the compacted files be cleaned up slower than normal(10min instead of 2min). Because when I run testCFKeyRotation() for 100 times, I have met assertion fail at `List<Path> compactedPaths = findCompactedStorefilePaths(htd.getTableName());
    assertTrue(compactedPaths.size() > 0);`. So this test case may also be flaky at this assertion. 
Enlarging the compaction discharger interval can ensure compacted files still be there after major compaction is completed but compacted paths are not checked in testCFKeyRotation().

Reply:
How about a bigger value? The test will timeout if it can't finished after 780 seconds. So may be use 30 * 60 * 1000 is better which is bigger than 2 * 780?

Reply:
I checked the test class config, it's a MediumTests.class, which will complete in 50 seconds. So the test method testCFKeyRotation() must complete in 50 secs, less than the cleaner interval(2 mins). I met this problem because when I ran 100 times of the method, I used parameterized running. All 100 cases used the same minicluster and when ran after 2 mins the case may met with checking compacted files fail.
@infraio I will convert the change of the conf.